rllib/tuned_examples/dqn/benchmark_dqn_atari.py

```diff
@@ -0,0 +1,366 @@
+import gymnasium as gym
+from gymnasium.wrappers import AtariPreprocessing
+
+from ray.rllib.algorithms.dqn.dqn import DQNConfig
+from ray.rllib.connectors.env_to_module.frame_stacking import FrameStackingEnvToModule
+from ray.rllib.connectors.learner.frame_stacking import FrameStackingLearner
+from ray.rllib.env.single_agent_env_runner import SingleAgentEnvRunner
+from ray.tune import Stopper
+from ray import train, tune
+
+# Might need `gymnasium[atari, other]` to be installed.
+
+# See the following links for becnhmark results of other libraries:
+#   Original paper: https://arxiv.org/abs/1812.05905
+#   CleanRL: https://wandb.ai/cleanrl/cleanrl.benchmark/reports/Mujoco--VmlldzoxODE0NjE
+#   AgileRL: https://github.com/AgileRL/AgileRL?tab=readme-ov-file#benchmarks
+
+benchmark_envs = {
```

Comment:
Awesome PR @simonsays1980 ! Did we actually run these on RLlib and got these results? Or how did you get to these episode-return numbers?

Reply:
Thanks ;) No we did not - I collected results from the papers. Take a look at my slack comment towards it, too.